app/boards/shields/stream_cheap/stream_cheap_2x4.keymap

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
This file should probably be named `stream_cheap.keymap`, it currently can't find the keymap

Reply:
hmm you dont get the keymap when you compile it with 
west build -p -b nice_nano_v2 -- -DSHIELD=stream_cheap_2x4
? If not we can go ahead and rename it all to just stream_cheap and if anyone decides to make the other variants they can change things around.